src/arc.directive.ts

```diff
@@ -57,22 +64,22 @@ export class ArcDirective implements OnInit, OnDestroy, IArcDirective {
   public arcBack = new EventEmitter<IArcEvent>();
 
   @Input('arc-left')
-  public set arcLeft(target: Element) {
+  public set arcLeft(target: any) {
     this.handlers.push(createDirectionCapture(Direction.LEFT, target));
   }
 
   @Input('arc-right')
-  public set arcRight(target: Element) {
+  public set arcRight(target: any) {
     this.handlers.push(createDirectionCapture(Direction.RIGHT, target));
   }
 
   @Input('arc-up')
-  public set arcUp(target: Element) {
+  public set arcUp(target: any) {
     this.handlers.push(createDirectionCapture(Direction.UP, target));
   }
 
   @Input('arc-down')
-  public set arcDown(target: Element) {
+  public set arcDown(target: any) {
```

Comment:
Please avoid using `any` types! Maybe you want `Element | HTMLElement | ElementRef`?